analyzers/tests/SonarAnalyzer.TestFramework/Verification/IssueValidation/IssueLocationCollector.cs

```diff
@@ -119,103 +117,93 @@ public static IList<IssueLocation> MergeLocations(IssueLocation[] locations, Lis
             return Enumerable.Empty<IssueLocation>();
         }
 
+        // ToDo: Throw away
         private static IEnumerable<IssueLocation> GetBuildErrorsLocations(string filePath, TextLine line) =>
-            GetLocations(filePath, line, RxBuildError);
+            FindLocations(filePath, line, RxBuildError);
 
-        private static IEnumerable<IssueLocation> GetLocations(string filePath, TextLine line, Regex rx)
+        // ToDo: Merge with its invocation site
+        private static IEnumerable<IssueLocation> FindLocations(string filePath, TextLine line, Regex rx)
```

Comment:
You'll see in the next PR. After removal of the error entry point, `FindLocations` will be called only once. So it will not need to be extracted anymore